textext.py

```diff
@@ -369,7 +389,7 @@ def do_convert(self, text, preamble_file, scale_factor, converter_class, old_nod
             self.settings.set('preamble', '')
 
         if scale_factor is not None:
-            self.settings.set('scale', scale_factor)
+            self.settings.set('scale', scale_factor_from_user)
```

Comment:
Then this would also maybe be called user_scale_factor instead.  But thats just a matter of taste anyway.

Reply:
see above